modules/admission-webhook-about.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-about_{context}"]
+= Admission webhooks
+
+In addition to {product-title} default admission controller plug-ins, admission webhooks enable you to introduce admission plug-ins invoked from webhook servers over HTTP at defined endpoints.
+
+There are two types of configurable admission webhooks in {product-title}:
+
+//Future xref - * During the admission process, a mutating admission controller can invoke xref:../architecture/admission-controllers.adoc#mutating-admission-webhooks[mutating admission webhooks] to perform tasks, such as injecting affinity labels.
+* During the admission process, a mutating admission controller can invoke mutating admission webhooks to perform tasks, such as injecting affinity labels.
+
+//Future xref - * At the end of the admission process, a validating admission controller can invoke xref:../architecture/admission-controllers.adoc#validating-admission-webhooks[validating admission webhooks] to make sure an object is configured properly, such as validating affinity labels. If the validation passes, {product-title} schedules the object as configured.
+* At the end of the admission process, a validating admission controller can invoke validating admission webhooks to make sure an object is configured properly, such as validating affinity labels. If the validation passes, {product-title} schedules the object as configured.
+
+When an API request comes in, a mutating or validating admission controller uses the list of external webhooks in the configuration and calls them in parallel:
+
+* If all of the webhooks approve the request, the admission chain continues.
+
+* If any of the webhooks deny the request, the admission request is denied and the reason for doing so is based on the first denial.
+
+* If more than one webhook denies the admission request, only the first is returned to the user.
```

Comment:
the first of what?

Reply:
The first denial reason. I have updated the document to reflect this. Please let me know if this is correct.